test/e2e/tests/phishing-controller/mocks.js

```diff
@@ -158,10 +167,8 @@ function resolveProviderConfigName(providerName) {
   switch (providerName.toLowerCase()) {
```

Comment:
This seems a bit redundant now. Did you keep it in case we want to add another provider again in the future?

Reply:
I did - I left it in case we needed to add a new provider in the future.  Let me know if you think thats okay or if you would like for me to remove it.

Reply:
That's fine :+1: